### PR TITLE
PAGOPA-406 add integration testing

### PR DESCRIPTION
**Pull request by cap-ang**
#### List of Changes

- test CRUD operation API
- tested scenarios: ok scenario 200, ko scenario 404, 409
- step definitions
- client to call this microservice
- configuration like environment variables setting

#### Motivation and Context

Integration testing adaptation from pagopa/pagopa-reporting-orgs-enrollment to pagopa/pagopa-poc-micronaut for both services (one based on h2 the other on azure-data-tables).

Language and tool: js and cucumber

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: integration-test/src/features/organizations.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Feature: All about Organizations
+
+  Background:
+    Reporting Enrollment running
+
+  Scenario: An organization creates an enrollment
+    Given a random id organization
+    When the organization "creates" enrollment
```
Comment (pasqualespica) [inline integration-test/src/features/organizations.feature:8]: i'm not sure but `"` are needed for string ?

File: integration-test/src/features/organizations.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  Scenario: An organization gets all the enrollments
+  	Given a valid enrollment
+  	When the organization "gets all" enrollment
+  	Then the organization gets the status code 200
+
+# 404
+
+  Scenario: An organization deletes an enrollment that does not exist
+    Given a random id organization
```
Comment (pasqualespica) [inline integration-test/src/features/organizations.feature]: In this case it's bettere use a specific unk id org ex. `UNKNOW`

File: integration-test/src/features/organizations.feature [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    When the organization "deletes" enrollment
+    Then the organization gets the status code 404
+
+  Scenario: An organization get an enrollment that does not exist
+  	Given a random id organization
```
Comment (pasqualespica) [inline integration-test/src/features/organizations.feature]: ditto

**Review by pasqualespica — CHANGES_REQUESTED** [review]

File: integration-test/src/features/organizations.feature (hunk 1 above)
Comment (cap-ang) [reply integration-test/src/features/organizations.feature:8]: Yes, matches single-quoted or double-quoted strings, ref: https://github.com/cucumber/cucumber-expressions#readme

**Review by pasqualespica — APPROVED** [review]

**Pull request merged**